examples/lustre/test.bats

```diff
@@ -0,0 +1,37 @@
+load "${CHTEST_DIR}/common.bash"
+
+setup () {
+    scope standard
+    if [[ -z "${ch_lustre}" ]]; then
+        skip 'no lustre to bind mount'
```

Comment:
I think the logic should be:
1. If not in a Slurm allocation and no Lustre dir, then just skip.
2. If in a Slurm allocation and no Lustre dir, then `pedantic_fail`, to force the team to run the Lustre tests.